packages/twenty-front/src/modules/object-record/record-field/states/recordPositionInternalState.ts

```diff
@@ -0,0 +1,6 @@
+import { createState } from 'twenty-ui';
+
+export const recordPositionInternalState = createState<number | null>({
```

Comment:
Position is confusing can we rename to rank (or rankInView)

Reply:
agree + same comment as aboved